helm/hpcc/values.yaml

```diff
@@ -63,6 +63,21 @@ global:
   #   postJobCommand: "kill $(pgrep linkerd2-proxy)"
   #   postJobCommandViaSidecar: true
 
+  ## visibilities section can be used to set labels, annotations and service type for any service with the specified visibility
+  visibilities:
+    cluster:
+      type: ClusterIP
+    local:
+      annotations:
+        service.beta.kubernetes.io/azure-load-balancer-internal: "true"
```

Comment:
Should this be commented out by default?

Reply:
No - because we want by default the system to work on azure without exposing things to the internet (thus keeping Flavio happy). It will be harmless on non-azure systems.